### PR TITLE
Upgrade documentation for v3

### PR DESCRIPTION
**Pull request by xPaw**
Let's merge it since new docker repo already uses the readme, and it's outdated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,18 +35,18 @@ or starting a container manually:
 $ docker run --detach \
 		--name thelounge \
 		--publish 9000:9000 \
-		--volume ~/.thelounge:/home/lounge/data \
+		--volume ~/.thelounge:/var/opt/thelounge \
 		--restart always \
-		thelounge/lounge:latest
+		thelounge/thelounge:3.0.0-pre.7
```
Comment (xPaw) [inline README.md]: This should remain `latest`?

File: README.md [hunk 2]
```diff
@@ -74,23 +55,23 @@ $ docker exec -it <container_name> thelounge add <username>
...
 change the host port in the port mapping. To make The Lounge available on e.g. port 5000:
 ```sh
 $ docker run --detach \
 		--name thelounge \
 		--publish 5000:9000 \ # Change host port to listen on port 5000
-		--volume ~/.thelounge:/home/lounge/data \
+		--volume ~/.thelounge:/var/opt/thelounge \
 		--restart always \
-		thelounge/lounge:latest
+		thelounge/thelounge:3.0.0-pre.7
```
Comment (xPaw) [inline README.md]: Same here.

File: README.md (hunk 1 above)
Comment (williamboman) [reply README.md]: yep! but latest doesn't exist on the new repo yet, will be created when 3.0.0 is released

**Comment by williamboman** [comment]
> Let's merge it since new docker repo already uses the readme, and it's outdated.

Yeah it needs to be manually updated, I must've forgotten to do it last build

File: README.md (hunk 1 above)
Comment (xPaw) [reply README.md]: Well it's either that, or hardcoded outdated version.

Comment (williamboman) [reply README.md]: My idea was to keep the `master` documentation refer to the latest stable release (2.7.1) and have this branch be sort of wip until 3.0.0 is released. There's also a [link](https://github.com/thelounge/thelounge-docker#v3-pre-releases) to the v3 docs in the 2.7.1 readme file

**Comment by xPaw** [comment]
I thought Docker directly pulled from git readme, but seeing as you have to manually update it, it's less of a problem.

**Comment by williamboman** [comment]
> I thought Docker directly pulled from git readme, but seeing as you have to manually update it, it's less of a problem.

It does, but only during building.

**Comment by williamboman** [comment]
> It was actually supposed to be all on the same line, not a list.

Oh, like so?

```diff
diff --git a/README.md b/README.md
index 67e194c..643ecba 100644
--- a/README.md
+++ b/README.md
@@ -2,9 +2,7 @@
 
 #### Docker container for The Lounge, modern web IRC client designed for self-hosting
 
--   [Website](https://thelounge.chat/)
--   [Docs](https://thelounge.chat/docs)
--   [Demo](https://demo.thelounge.chat/)
+[Website](https://thelounge.chat/) • [Docs](https://thelounge.chat/docs) • [Demo](https://demo.thelounge.chat/)
 
 [![#thelounge IRC channel on freenode"](https://img.shields.io/badge/freenode-%23thelounge-415364.svg?colorA=ff9e18&style=flat-square)](https://demo.thelounge.chat/) [![Total pulls on Docker](https://img.shields.io/docker/pulls/thelounge/thelounge.svg?style=flat-square)](https://hub.docker.com/r/thelounge/thelounge/)
 

```

**Comment by xPaw** [comment]
Yep but keep \*\*links\*\* around for bold.

**Comment by xPaw** [comment]
@williamboman Merging this and I will publish another RC today, so we can the new docker version with new docs.

**Comment by williamboman** [comment]
We need to bump build args in the Dockerfiles and push a new tag for DockerHub to catch on. You can imitate my release PRs and do it yourself if you feel like it, otherwise Ill get around to it later today

**Comment by xPaw** [comment]
I'll leave it up to you.

**Pull request merged**